StudioCore/ParamEditor/ParamEditorScreen.cs

```diff
@@ -1229,7 +1257,7 @@ public class ParamEditorSelectionState
         private string _activeParam = null;
         private Dictionary<string, ParamEditorParamSelectionState> _paramStates = new Dictionary<string, ParamEditorParamSelectionState>();
 
-        public bool paramSelectionExists()
+        public bool activeParamExists()
```

Comment:
Is there much reason for this refactor? It does for the moment capture the data better (since only 1 param can be selected)
Just watch out for merge conflicts

Reply:
"selected" and "active" were being used interchangably in the file, but mostly "active", so I corrected an instance of "selected"